zitadel/v2/application_oidc/funcs.go

```diff
@@ -63,10 +63,14 @@ func update(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Dia
 		}
 	}
 
-	if d.HasChanges(redirectURIsVar,
+	if d.HasChanges(
+		redirectURIsVar,
+		responseTypesVar,
+		grantTypesVar,
 		appTypeVar,
 		authMethodTypeVar,
 		postLogoutRedirectURIsVar,
+		versionVar,
```

Comment:
Let's force recreation, as UpdateOIDCAppConfig doesn't support the version field.

```suggestion
```

Instead, it additionally supports the field SkipNativeAppSuccessPage.